rest/taskana-rest-spring/src/main/java/pro/taskana/rest/ClassificationController.java

```diff
@@ -76,6 +68,15 @@
         }
     }
 
+    @GetMapping(path = "/domains")
+    @Transactional(readOnly = true, rollbackFor = Exception.class)
+    public ResponseEntity<List<String>> getDomains() {
+        List<String> export = new ArrayList<>();
+        ClassificationQuery classificationQuery = classificationService.createClassificationQuery();
```

Comment:
I am not sure if createClassificationQuery can trigger exceptions, make sure it won't trigger them. In other case use try catch.